t/35-script_clone_job.t

```diff
@@ -56,6 +57,7 @@ my %parent_settings = (
     HDDSIZEGB    => 20,
     WORKER_CLASS => 'qemu_x86_64',
 );
+my $dir = tempdir("/tmp/$FindBin::Script-XXXX")->make_path;
```

Comment:
you tend to reuse this line as well. As you have a good idea why you want to have an explicitly named tempdir (and I forgot what it was) I recommend to also pull this one out into a nicely named helper function so that already from the function name we know what this should ensure